javamelody-core/src/main/java/net/bull/javamelody/Parameter.java

```diff
@@ -202,7 +202,7 @@
 	SYSTEM_ACTIONS_ENABLED("system-actions-enabled"),
 
 	/**
-	 * Active la protection contre CSRF (false par défaut).
+	 * Parameter to enable or disable the Content Security Policy header (true by default).

```

Comment:
This is not the right place for this javadoc.

Reply:
Fixed!